apollo-client/src/test/java/com/ctrip/framework/apollo/internals/AbstractConfigTest.java

```diff
@@ -71,7 +71,7 @@ public void onChange(ConfigChangeEvent changeEvent) {
     // wait a minute for invoking
     Thread.sleep(100);
 
-    verify(configChangeListener, times(0)).onChange(Matchers.<ConfigChangeEvent>any());
+    verify(configChangeListener, times(0)).onChange(ArgumentMatchers.any());
```

Comment:
remove the class identifier to forbid it will change again in the feature?
```suggestion
    verify(configChangeListener, times(0)).onChange(any());
```
add import static
```java
import static org.mockito.ArgumentMatchers.any;
```

Reply:
good idea